games-roguelike/dwarf-fortress/dwarf-fortress-0.47.05.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic optfeature prefix toolchain-funcs
+
+MY_PV=$(ver_rs 0- _ "$(ver_cut 2-)")
+MY_PN=df
+MY_P=${MY_PN}_${MY_PV}
+
+DESCRIPTION="The deepest, most intricate simulation of a world that's ever been created."
+HOMEPAGE="https://www.bay12games.com/dwarves"
+SRC_URI="amd64? ( https://www.bay12games.com/dwarves/${MY_P}_linux.tar.bz2 )
+	x86? ( https://www.bay12games.com/dwarves/${MY_P}_linux32.tar.bz2 )"
+
+LICENSE="free-noncomm BSD BitstreamVera"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+IUSE="debug"
+
+RDEPEND="media-libs/glew:0
+	media-libs/libsdl[joystick,video]
+	media-libs/sdl-image[png]
+	media-libs/sdl-ttf
+	sys-libs/zlib
+	virtual/glu
+	x11-libs/gtk+:2"
+
+# Yup, libsndfile, openal and ncurses are only needed at compile-time; the code
+# dlopens them at runtime if requested.
```

Comment:
But it'll then fail at runtime if requested and the library is not there?

Reply:
Good question! I've actually been meaning to test this - I think it just disables the option gracefully, but I'll double check.